app/components/post_draft/send_handler/send_handler.tsx

```diff
@@ -226,7 +190,7 @@ export default function SendHandler({
         if (data?.goto_location && !value.startsWith('/leave')) {
             handleGotoLocation(serverUrl, intl, data.goto_location);
         }
-    }, [userIsOutOfOffice, currentUserId, intl, value, serverUrl, channelId, rootId, handleEndCall]);
+    }, [userIsOutOfOffice, currentUserId, intl, value, serverUrl, channelId, rootId, handleCallsSlashCommand]);
```

Comment:
`handleCallsSlashCommand` should not be needed as a dependency, as it is an imported function, and therefore we don't expect it to change during the lifetime of the build.